nutanix/resource_nutanix_ndb_database.go

```diff
@@ -717,6 +781,123 @@ func expandActionArguments(d *schema.ResourceData) []*era.Actionarguments {
 					Value: values,
 				})
 			}
+
+			if ha, ok := val["ha_availability"]; ok && len(ha.([]interface{})) > 0 {
+				haList := ha.([]interface{})
+
+				for _, v := range haList {
+					val := v.(map[string]interface{})
+					var values interface{}
+
+					if haProxy, pok := val["proxy_read_port"]; pok && len(haProxy.(string)) > 0 {
+						values = haProxy
+
+						args = append(args, &era.Actionarguments{
+							Name:  "proxy_read_port",
+							Value: values,
+						})
+					}
+
+					if proxyWrite, pok := val["proxy_write_port"]; pok && len(proxyWrite.(string)) > 0 {
+						values = proxyWrite
+
+						args = append(args, &era.Actionarguments{
+							Name:  "proxy_write_port",
+							Value: values,
```

Comment:
Directly we can assign. Can we change in all 
```suggestion
							Value: proxyWrite,
```

Reply:
Done.